scripts/ci-entrypoint.sh

```diff
@@ -199,9 +206,8 @@ wait_for_pods() {
 }
 
 install_addons() {
-    until copy_kubeadm_config_map; do
-        sleep 5
-    done
+    export -f copy_kubeadm_config_map wait_for_copy_kubeadm_config_map
+    timeout --foreground 600 bash -c wait_for_copy_kubeadm_config_map
```

Comment:
Are we exporting both the functions here because the compiler complains if we only export `wait_for_copy_kubeadm_config_map` ? 

Also, why are we exporting `wait_for_copy_kubeadm_config_map` here? Are there any advantages of making this function available to other child processes via export?